docker-compose.yml

```diff
@@ -1,19 +1,5 @@
 version: '3.1'
 services:
```

Comment:
Altered so that locally running prison-register will work with postgres db and auth service.
Question: Should the compose file be and local profile include token verification?

Reply:
The other apis don't tend to so I'd leave it out